tensorboard/plugins/interactive_inference/witwidget/notebook/base.py

```diff
@@ -16,10 +16,10 @@
 import json
 import googleapiclient.discovery
```

Comment:
Should probably do this as a separate PR but how was this previously working? GAPI isn't part of our dependency declaration and I believe we didn't do anything special to install it.

Reply:
acknowledged